opam.export

```diff
@@ -1,5 +1,7 @@
 opam-version: "2.0"
+compiler: ["ocaml-base-compiler.4.14.0"]
```

Comment:
This change broke the makefile for me (`check_opam_switch` fails). Do you mind if I revert it?

Reply:
(reverted it)

Reply:
you mean this line or this file changing? This file changing is to add a needed OCaml dependency  :o

Reply:
looking at your change it seems like it was that line, weird that it introduced an issue for you but not for me : o